FIAT/__init__.py

```diff
@@ -50,7 +50,6 @@
                       "Discontinuous Raviart-Thomas": DiscontinuousRaviartThomas,
                       "Hermite": CubicHermite,
                       "Lagrange": Lagrange,
-                      "Gauss-Lobatto-Legendre": GaussLobattoLegendre,
```

Comment:
Right, you did the above because of this, but why do you want to remove this? Maybe sloppy rebase/merge?